### PR TITLE
Mika home task: HTML_01

### PR DESCRIPTION
**Pull request by mikagolubenkov**
My home task: HTML_01

### CODE REVIEW

File: mgolubenkov/01_HTML/html0.html [hunk 1]
```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
```
Comment (pluhin) [inline mgolubenkov/01_HTML/html0.html]: This task wasn't been finished

File: mgolubenkov/01_HTML/html1.html [hunk 2]
```diff
@@ -0,0 +1,66 @@
+<!DOCTYPE html>
+	<HTML> 
+		<HEAD>
+			<META charset="utf-8">
+			<TITLE>Пример страницы</TITLE>
+		</HEAD>
+		<BODY>
+			<H1>HOW-TO: Программа на Python, Ча   ghhhghg     сть 1</H1>
```
Comment (pluhin) [inline mgolubenkov/01_HTML/html1.html]: extra text in header

File: mgolubenkov/01_HTML/html1.html [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+			<P>	<DEL>Добрый день,
```
Comment (pluhin) [inline mgolubenkov/01_HTML/html1.html]: What does it doing here?

File: mgolubenkov/01_HTML/html1.html [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+			<TITLE>Пример страницы</TITLE>
+		</HEAD>
+		<BODY>
+			<H1>HOW-TO: Программа на Python, Ча   ghhhghg     сть 1</H1>
+			<P>	<DEL>Добрый день, 
+				<EM>Уважаемые слушатели</EM>
+				</DEL>
+		
+		
+ Программа на Python — часть 1
```
Comment (pluhin) [inline mgolubenkov/01_HTML/html1.html]: This task is not finished

File: mgolubenkov/01_HTML/html12.html [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+		<HEAD>
+			<META charset="utf-8">
+			<TITLE>Программа на Python, Часть 1</TITLE>
+		</HEAD>
+		<BODY> 
+			<H1>HOW-TO: Программа на Python, Часть 1</H1>
+			<Ul>
+				<li><DEL>Программа на Python — часть 1</DEL></LI>
+				<li><DEL>Программа на Python — часть 2</DEL></li>
+				<li>Программа на Python — часть 3</li>
```
Comment (pluhin) [inline mgolubenkov/01_HTML/html12.html]: you did not close ul

File: mgolubenkov/01_HTML/html12.html [hunk 6]
```diff
@@ -0,0 +1,46 @@
...
+			<h3>HOW-TO: Программа на Python, Часть 1</h3>
+			<ul>
+				<li>Наша первая программа</li>
+				<li>Пример цикла For</li>
+			</ul>
+			<p>Из множества существующих языков программирования Python — один из самых лёгких в изучении.</p>
+			<p>Он создан в конце 80-х годов XX века, но с тех пор он сильно изменился. Хотя Python входит в состав большинства дистрибутивов Linux, на него часто не обращают внимания при выборе языкадля изучения. В этой статье мы рассмотрим создание консольных приложений, а в следующей —программирование графических интерфейсов (GUI). Давайте создадим простую программу прямо сейчас.
+			<h2>Наша первая программа</h2>
+			<p>Наберём код в текстовом редакторе (например, в gedit). Потом мы разберёмся, что делает каждая строка и пойдём дальше.
+			<h4>Введите эти 4 строки:</h4>
```
Comment (pluhin) [inline mgolubenkov/01_HTML/html12.html]: why h4? should be h2

File: mgolubenkov/01_HTML/html13.html [hunk 7]
```diff
@@ -0,0 +1,49 @@
...
+			<p>Это всё. Сохраните файл как <b>hello.py</b> в любом месте. Я предлагаю сохранить его в папке <b>python_examples</b> в домашнем каталоге. Этот пример показывает, как просто написать программу на Python. Для запуска программы нужно сделать её файл исполняемым. Для этого введите в терминале
+			<p><ins><i><b>chmod +x hello.py</b></i></ins></p>
+			<p>в папке, в которой вы сохранили программу. Теперь запустите её.</p>
+			<p></p>
+<pre>greg@earth:~/python_examples$ ./hello.py
+Hello. I am a python program.
+What is your name? Ferd Burphel
+Hello there, Ferd Burphel!
+greg@earth:~/python_examples$</pre>
+			<p></p>
```
Comment (pluhin) [inline mgolubenkov/01_HTML/html13.html]: Do not use empty paragraphs

File: mgolubenkov/01_HTML/html2.html [hunk 8]
```diff
@@ -0,0 +1,66 @@
+<!DOCTYPE html>
+	<HTML> 
+		<HEAD>
+			<META charset="utf-8">
+			<TITLE>Пример 123 страницы</TITLE>
+		</HEAD>
+		<BODY>
+			<H1>HOW-TO: Программа на Python, Часть 222221</H1>
+			<P>	<DEL>Добрый день,
```
Comment (pluhin) [inline mgolubenkov/01_HTML/html2.html]: remove this text it isn't related to this particular task

File: mgolubenkov/01_HTML/html2.html [hunk 9]
```diff
@@ -0,0 +1,66 @@
...
+			<TITLE>Пример 123 страницы</TITLE>
+		</HEAD>
+		<BODY>
+			<H1>HOW-TO: Программа на Python, Часть 222221</H1>
+			<P>	<DEL>Добрый день, 
+				<EM>Уважаемые слушатели</EM>
+				</DEL>
+		
+		
+ Программа на Python — часть 1
```
Comment (pluhin) [inline mgolubenkov/01_HTML/html2.html]: Not finished

File: mgolubenkov/02_HTML/my_CALENDAR.html [hunk 10]
```diff
@@ -0,0 +1,168 @@
...
+		<!-- Значения
+			void		Не отрисовывать границы.
+			border		Граница вокруг таблицы.
+			above		Граница по верхнему краю таблицы.
+			below		Граница снизу таблицы.
+			hsides		Добавить только горизонтальные границы (сверху и снизу таблицы).
+			vsides		Рисовать только вертикальные границы (слева и справа от таблицы).
+			rhs			Граница только на правой стороне таблицы.
+			lhs			Граница только на левой стороне таблицы.-->
+  		<table border="1" align="center" width="14%" frame="hsides" cellpadding="5">
```
Comment (pluhin) [inline mgolubenkov/02_HTML/my_CALENDAR.html]: do not use and remember old attributes like frame and cellpadding

File: mgolubenkov/02_HTML/my_CALENDAR.html [hunk 11]
```diff
@@ -0,0 +1,168 @@
...
+   		<caption><b>My manth MARCH</b></caption>
+   		<tr>
+    		<th colspan="6"><b>Январь</b></th>
```
Comment (pluhin) [inline mgolubenkov/02_HTML/my_CALENDAR.html]: b -not needed here

Comment (pluhin) [reply mgolubenkov/02_HTML/my_CALENDAR.html]: as in  all tags th - it already implemented in specification of this tag

File: mgolubenkov/02_HTML/my_CALENDAR.html [hunk 12]
```diff
@@ -0,0 +1,168 @@
...
+   			<td>  </td>
+   		</tr>
+ 		</table>
+ 	</body>
+</html>
+
+
+
+
+<html>
```
Comment (pluhin) [inline mgolubenkov/02_HTML/my_CALENDAR.html]: since here should be next file

File: mgolubenkov/01_HTML/html0.html (hunk 1 above)
Comment (mikagolubenkov) [reply mgolubenkov/01_HTML/html0.html]: excuse me of me, it is rough copy file

File: mgolubenkov/01_HTML/html1.html (hunk 4 above)
Comment (pluhin) [reply mgolubenkov/01_HTML/html1.html]: Then it needs to remove from PR

File: mikagolubenkov/04_HTML/forms1.html [hunk 13]
```diff
@@ -0,0 +1,169 @@
...
+            	<th><b>ПН</b></th>
+			<!--02-12-->
+                <td>  </td>
+            </tr>
+        </table>	
+    	</BODY>
+    </HTML> ​
+    
+
+ <!--   <!DOCTYPE html>
```
Comment (pluhin) [inline mikagolubenkov/04_HTML/forms1.html]: Could you please remove this comment

File: mikagolubenkov/04_HTML/forms1.html [hunk 14]
```diff
@@ -0,0 +1,169 @@
...
+                        <label> 
+                            <input type="radio" name="search_dist" value="google">
+                            в Google
+                        </label><br>
+						<i>Длина запроса не более 10 символов</i>
+                </fieldset>
+				</td>
+				<!--01-12-->
+				<td>
+				<b>Простая форма поиска с вариантами:</b><br>
```
Comment (pluhin) [inline mikagolubenkov/04_HTML/forms1.html]: Here should be header

File: mikagolubenkov/04_HTML/forms1.html [hunk 15]
```diff
@@ -0,0 +1,169 @@
...
+               <th colspan="6"><b>Март</b></th>
+            </tr>-->
+			<!--Ferst -->
+			<!--<tr>
+                <th><b>Простая форма поиска с вариантами:</b></th>
+                <th><b>Простая форма поиска с вариантами:</b></th><br>
+            </tr>-->
+            <tr>
+                <td>
+				<b>Простая форма поиска с вариантами:</b><br>
```
Comment (pluhin) [inline mikagolubenkov/04_HTML/forms1.html]: Here should be header

File: mikagolubenkov/04_HTML/forms1.html [hunk 16]
```diff
@@ -0,0 +1,169 @@
...
+				</td>
+				<!--01-12-->
+				<td>
+				<b>Простая форма поиска с вариантами:</b><br>
+				<fieldset>
+                    <legend>Строка поиска</legend>
+					<input type="text" name="search" maxlength="10" required autofocus>
+					<span>*</span>
+					<input type="hidden" name="hidden_field" value="hidden">
+					<input type="submit" value="Искатьд"><br>
```
Comment (pluhin) [inline mikagolubenkov/04_HTML/forms1.html]: Typo

File: mikagolubenkov/04_HTML/forms1.html [hunk 17]
```diff
@@ -0,0 +1,169 @@
+<!DOCTYPE html>
```
Comment (pluhin) [inline mikagolubenkov/04_HTML/forms1.html]: This task hasn't been done

File: mikagolubenkov/03_HTML/Piton01.html [hunk 18]
```diff
@@ -0,0 +1,66 @@
...
+			<META charset="utf-8">
+			<TITLE>main</TITLE>
+		</HEAD>
+		<BODY>
+			<H2>main.mtml</H2>
+			<P>	<DEL>Добрый день, 
+				<EM>Уважаемые слушатели</EM>
+				</DEL>
+		
+
```
Comment (pluhin) [inline mikagolubenkov/03_HTML/Piton01.html]: This task hasn't been done

File: mikagolubenkov/02_HTML/prognoz_POGODY.html [hunk 19]
```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML  4.01//EN" "http://www.w3.org/TR/html4/strict.dtd">
```
Comment (pluhin) [inline mikagolubenkov/02_HTML/prognoz_POGODY.html]: we use html 5, here - 4 version

File: mikagolubenkov/02_HTML/prognoz_POGODY.html [hunk 20]
```diff
@@ -0,0 +1,43 @@
...
+			above		Граница по верхнему краю таблицы.
+			below		Граница снизу таблицы.
+			hsides		Добавить только горизонтальные границы (сверху и снизу таблицы).
+			vsides		Рисовать только вертикальные границы (слева и справа от таблицы).
+			rhs			Граница только на правой стороне таблицы.
+			lhs			Граница только на левой стороне таблицы.-->
+  		<table border="1" align="center" width="30%" frame="hsides" cellpadding="5">
+   		<caption><i>Прогноз погоды</i></caption>
+   		<tr>
+    		<th><b>Город</b></th>
```
Comment (pluhin) [inline mikagolubenkov/02_HTML/prognoz_POGODY.html]: remove b, tag th already provides bolder

File: mikagolubenkov/02_HTML/prognoz_POGODY.html [hunk 21]
```diff
@@ -0,0 +1,43 @@
...
+    		<th><b>	&#8451;Утро	&#176;</b></th>
```
Comment (pluhin) [inline mikagolubenkov/02_HTML/prognoz_POGODY.html]: remove b, tag th already provides bolder

File: mikagolubenkov/02_HTML/prognoz_POGODY.html [hunk 22]
```diff
@@ -0,0 +1,43 @@
...
+   		</tr>
+   		<tr>
+    		<th><b>Минск</b></th>
+    		<td>-5	&#8451;</td>
+   			<td>+9	&#176;</td>
+    		<td rowspan="2">+2	&#176;</td>
+   			<td>-10	&#176;</td>
+   		</tr>
+   		<tr>
+    		<th><b>Брест</b></th>
```
Comment (pluhin) [inline mikagolubenkov/02_HTML/prognoz_POGODY.html]: remove b, tag th already provides bolder

File: mikagolubenkov/02_HTML/prognoz_POGODY.html [hunk 23]
```diff
@@ -0,0 +1,43 @@
...
+   		<caption><i>Прогноз погоды</i></caption>
+   		<tr>
+    		<th><b>Город</b></th>
+    		<th><b>	&#8451;Утро	&#176;</b></th>
+   			<th><b>День</b></th>
+   			<th><b>Вечер</b></th>
+   			<th><b>Ночь</b></th>
+   		</tr>
+   		<tr>
+    		<th><b>Минск</b></th>
```
Comment (pluhin) [inline mikagolubenkov/02_HTML/prognoz_POGODY.html]: remove b, tag th already provides bolder

File: mikagolubenkov/02_HTML/my_CALENDAR.html [hunk 24]
```diff
@@ -0,0 +1,82 @@
+<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML  4.01//EN" "http://www.w3.org/TR/html4/strict.dtd">
```
Comment (pluhin) [inline mikagolubenkov/02_HTML/my_CALENDAR.html]: we use html5 here - html4

File: mgolubenkov/01_HTML/Python1.html [hunk 25]
```diff
@@ -0,0 +1,48 @@
...
+		<HEAD>
+			<META charset="utf-8">
+			<TITLE>Программа на Python, Часть 1</TITLE>
+		</HEAD>
+		<BODY> 
+			<H1>HOW-TO: Программа на Python, Часть 1</H1>
+			<a herf="#chap_1">Программа на Python — часть 1</a> <br>
+			<a herf="#chap_2">Программа на Python — часть 2</a> <br>
+			<a herf="#chap_3">Программа на Python — часть 3</a> <br>
+UUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUU
```
Comment (pluhin) [inline mgolubenkov/01_HTML/Python1.html]: what is it?

File: mgolubenkov/02_HTML/prognoz_POGODY.html [hunk 26]
```diff
@@ -0,0 +1,43 @@
...
+			hsides		Добавить только горизонтальные границы (сверху и снизу таблицы).
+			vsides		Рисовать только вертикальные границы (слева и справа от таблицы).
+			rhs			Граница только на правой стороне таблицы.
+			lhs			Граница только на левой стороне таблицы.-->
+  		<table border="1" align="center" width="30%" frame="hsides" cellpadding="5">
+   		<caption><i>Прогноз погоды</i></caption>
+   		<tr>
+    		<th><b>Город</b></th>
+    		<th><b>	&#8451;Утро	&#176;</b></th>
+   			<th><b>День</b></th>
```
Comment (pluhin) [inline mgolubenkov/02_HTML/prognoz_POGODY.html]: the th already has bold, need to remove tag b from th

File: mgolubenkov/03_HTML/meny1.html [hunk 27]
```diff
@@ -0,0 +1,105 @@
+<!DOCTYPE html>
```
Comment (pluhin) [inline mgolubenkov/03_HTML/meny1.html]: here two files in one, should be one

File: mgolubenkov/04_HTML/forms1.html [hunk 28]
```diff
@@ -0,0 +1,190 @@
...
+							<option value="Минск">Минск</option>
+							<option value="Витебск">Витебск</option>
+							<option selected value="Города России"><b>Города России</b></option>
+							<option value="Москва">Москва</option>
+							<option value="Рязень">Рязень</option>
+							<option value="Магадан">Магадан</option>   
+							</select>
+
+							<select name="list[]" size="8" multiple id="list">
+							<option value="1">Города РБ</option>
```
Comment (pluhin) [inline mgolubenkov/04_HTML/forms1.html]: Why this option are disabled?

File: mgolubenkov/04_HTML/forms1.html [hunk 29]
```diff
@@ -0,0 +1,190 @@
...
+				</fieldset>	
+				</td>
+			
+			<!--03-2-->
+			
+				<td>
+					<fieldset>
+						<legend>Города РБ</legend>
+						<select size="1" multiple name="hero[]">
+							<option selected value="Города РБ"><b>Города РБ</b></option>
```
Comment (pluhin) [inline mgolubenkov/04_HTML/forms1.html]: here should be optgroup tag

File: mgolubenkov/04_HTML/forms1.html [hunk 30]
```diff
@@ -0,0 +1,190 @@
...
+		<!--04-1-->
+			<tr>
+					<td><b>Загрузка файла на сервер:</b></td>
+					<td><b>Валидация данных HTML5:</b></td>
+			</tr>
+			<tr>
+					<td>
+					<fieldset>
+						<legend>Upload</legend>
+							Файл:<br>
```
Comment (pluhin) [inline mgolubenkov/04_HTML/forms1.html]: It should be label

File: mgolubenkov/04_HTML/forms1.html [hunk 31]
```diff
@@ -0,0 +1,190 @@
...
+				</td>
+			</tr>
+		<!--04-1-->
+			<tr>
+					<td><b>Загрузка файла на сервер:</b></td>
+					<td><b>Валидация данных HTML5:</b></td>
+			</tr>
+			<tr>
+					<td>
+					<fieldset>
```
Comment (pluhin) [inline mgolubenkov/04_HTML/forms1.html]: where is tag form here?

File: mgolubenkov/04_HTML/forms1.html [hunk 32]
```diff
@@ -0,0 +1,190 @@
...
+							enctype="multipart/form-data" для отправки
+							файла на сервер</i>
+					</fieldset>	
+					</td>
+		<!--04-2-->
+					<td>
+						<fieldset>
+							<legend>Строка поиска</legend>
+							<input type="text" name="name_03" pattern="[А-Я][а-я]+" placeholder="Только русские буквы"><br>
+							<i>Телефонный номер в формате<br>
```
Comment (pluhin) [inline mgolubenkov/04_HTML/forms1.html]: It should be label

File: mgolubenkov/04_HTML/forms1.html [hunk 33]
```diff
@@ -0,0 +1,190 @@
...
+							<input type="submit" value=" заргузить"><br>
+							<i>Метод POST<br>
+							В теге form используйте атрибут
+							enctype="multipart/form-data" для отправки
+							файла на сервер</i>
+					</fieldset>	
+					</td>
+		<!--04-2-->
+					<td>
+						<fieldset>
```
Comment (pluhin) [inline mgolubenkov/04_HTML/forms1.html]: where is tag form here?

File: mgolubenkov/04_HTML/forms1.html [hunk 34]
```diff
@@ -0,0 +1,190 @@
...
+							<legend>Строка поиска</legend>
+							<input type="text" name="name_03" pattern="[А-Я][а-я]+" placeholder="Только русские буквы"><br>
+							<i>Телефонный номер в формате<br>
+							+ХХХ-ХХ-ХХХХХХХ, где Х - любое числоы</i><br>
+							<input type="text" name="name_04" pattern="\+375-(29|44|25)-[0-9]{7}" placeholder="+375-29-1234567"><br>
```
Comment (pluhin) [inline mgolubenkov/04_HTML/forms1.html]: wrong pattern

File: mgolubenkov/04_HTML/forms1.html [hunk 35]
```diff
@@ -0,0 +1,190 @@
...
+						поле email - с проверкой<br>
+						метод POST
+						</i><br>
+				</fieldset>	
+				</td>
+			
+			<!--03-2-->
+			
+				<td>
+					<fieldset>
```
Comment (pluhin) [inline mgolubenkov/04_HTML/forms1.html]: whee is tag form here?

File: mgolubenkov/04_HTML/forms1.html [hunk 36]
```diff
@@ -0,0 +1,190 @@
...
+				<td><b>Выпадающий и мульти списки:</b></td>
+			</tr>
+			<tr>
+				<td>
+				<fieldset>
+					<legend>Оставьте Ваш отзыв</legend>
+						<input type="text" name="search" maxlength="10">
+						<span>*</span> Имя
+						<input type="text" name="search" maxlength="10">
+						<span>*</span> e-mail<br>
```
Comment (pluhin) [inline mgolubenkov/04_HTML/forms1.html]: here should be label

File: mgolubenkov/04_HTML/forms1.html [hunk 37]
```diff
@@ -0,0 +1,190 @@
...
+			<tr>
+            	<td><b>Отзыв:</b></td>
+				<td><b>Выпадающий и мульти списки:</b></td>
+			</tr>
+			<tr>
+				<td>
+				<fieldset>
+					<legend>Оставьте Ваш отзыв</legend>
+						<input type="text" name="search" maxlength="10">
+						<span>*</span> Имя
```
Comment (pluhin) [inline mgolubenkov/04_HTML/forms1.html]: here should be label

File: mgolubenkov/04_HTML/forms1.html [hunk 38]
```diff
@@ -0,0 +1,190 @@
...
+			<!--03-1-->
+			<tr>
+            	<td><b>Отзыв:</b></td>
+				<td><b>Выпадающий и мульти списки:</b></td>
+			</tr>
+			<tr>
+				<td>
+				<fieldset>
+					<legend>Оставьте Ваш отзыв</legend>
+						<input type="text" name="search" maxlength="10">
```
Comment (pluhin) [inline mgolubenkov/04_HTML/forms1.html]: duplicate name with line 106

File: mgolubenkov/04_HTML/forms1.html [hunk 39]
```diff
@@ -0,0 +1,190 @@
...
+				</td>
+			</tr>
+			<!--03-1-->
+			<tr>
+            	<td><b>Отзыв:</b></td>
+				<td><b>Выпадающий и мульти списки:</b></td>
+			</tr>
+			<tr>
+				<td>
+				<fieldset>
```
Comment (pluhin) [inline mgolubenkov/04_HTML/forms1.html]: where is form

File: mgolubenkov/04_HTML/forms1.html [hunk 40]
```diff
@@ -0,0 +1,190 @@
...
+					<i>* - обязательное поле</i><br>
+				</fieldset>	
+				<br>
+				<br>
+				</td>
+			<!--02-2-->
+				<td>
+				<fieldset>
+						<legend>Как вам?</legend>
+						<input type="radio" name="drink" value="rad1" checked>ничего<Br>
```
Comment (pluhin) [inline mgolubenkov/04_HTML/forms1.html]: each input and text here  should be in label

File: mgolubenkov/04_HTML/forms1.html [hunk 41]
```diff
@@ -0,0 +1,190 @@
...
+					<input type="checkbox" name="drink" value="rad1">Оставаться в системе<Br>
+					<input type="submit" value="Войти"><br>
+					<i>* - обязательное поле</i><br>
+				</fieldset>	
+				<br>
+				<br>
+				</td>
+			<!--02-2-->
+				<td>
+				<fieldset>
```
Comment (pluhin) [inline mgolubenkov/04_HTML/forms1.html]: where is form tag?

File: mgolubenkov/04_HTML/forms1.html [hunk 42]
```diff
@@ -0,0 +1,190 @@
...
+				<td><b>Опрос(голосование):</b></td>
+			</tr>
+			<tr>
+				<td>
+				<fieldset>
+                    <legend>Авторизируйтесь</legend>
+					<input type="text" name="search" maxlength="10">
+					<span>*</span> Логин
+					<input type="text" name="search" maxlength="10">
+					<span>*</span> Пароль<br>
```
Comment (pluhin) [inline mgolubenkov/04_HTML/forms1.html]: here should be label

File: mgolubenkov/04_HTML/forms1.html [hunk 43]
```diff
@@ -0,0 +1,190 @@
...
+			<tr>
+				<td><b>Простая форма авторизации:</b></td>
+				<td><b>Опрос(голосование):</b></td>
+			</tr>
+			<tr>
+				<td>
+				<fieldset>
+                    <legend>Авторизируйтесь</legend>
+					<input type="text" name="search" maxlength="10">
+					<span>*</span> Логин
```
Comment (pluhin) [inline mgolubenkov/04_HTML/forms1.html]: here should be label

File: mgolubenkov/04_HTML/forms1.html [hunk 44]
```diff
@@ -0,0 +1,190 @@
...
+				</td>
+			</tr>
+			<!--02-1-->
+			<tr>
+				<td><b>Простая форма авторизации:</b></td>
+				<td><b>Опрос(голосование):</b></td>
+			</tr>
+			<tr>
+				<td>
+				<fieldset>
```
Comment (pluhin) [inline mgolubenkov/04_HTML/forms1.html]: where is form tag

File: mgolubenkov/04_HTML/forms1.html [hunk 45]
```diff
@@ -0,0 +1,190 @@
...
+                    <legend>Авторизируйтесь</legend>
+					<input type="text" name="search" maxlength="10">
```
Comment (pluhin) [inline mgolubenkov/04_HTML/forms1.html]: duplicate name with line 72

File: mgolubenkov/04_HTML/forms1.html [hunk 46]
```diff
@@ -0,0 +1,190 @@
...
+					<span>*</span> Логин
+					<input type="text" name="search" maxlength="10">
```
Comment (pluhin) [inline mgolubenkov/04_HTML/forms1.html]: here should be type=password

**Pull request closed without merging**